src/forms/futureIncome.js

```diff
@@ -87,7 +87,7 @@ const FutureIncomeStep = function ( props ) {
     <Form className = 'income-form'>
       <FormPartsContainer
         title     = 'Future Household Income'
-        clarifier = 'How much money do you think your household could make in the future?'
+        clarifier = 'How much money would your household make in the future?'
```

Comment:
How come 'would' and not 'will' ?

Reply:
The discussion with Krisitin seemed to indicate that they would enter projected values so they could experiment to see what the effects would be on their benefits if certain factors of their situations changed.

Reply:
okay that makes sense.